actors/evm/src/interpreter/test_util.rs

```diff
@@ -13,13 +13,43 @@ macro_rules! evm_instruction {
 
 #[macro_export]
 macro_rules! evm_unit_test {
-    (($rt:ident, $machine:ident) { $($inst:tt;)* } $($body:tt)*) => {
+    (($machine:ident) [ $($inst:tt;)* ] $($body:tt)*) => {
+        use ::fil_actors_runtime::test_utils::MockRuntime;
+        use ::fvm_shared::econ::TokenAmount;
+        use $crate::interpreter::{execution::Machine, system::System, Output};
+        use $crate::{Bytecode, Bytes, EthAddress, ExecutionState};
+
+        let mut rt = MockRuntime::default();
+        let mut state = ExecutionState::new(
+            EthAddress::from_id(1000),
+            EthAddress::from_id(1000),
+            TokenAmount::from_atto(0),
+            Bytes::default(),
+        );
+
+        let code = vec![$($crate::evm_instruction!($inst)),*];
+
+        let mut system = System::new(&mut rt, false);
+        let bytecode = Bytecode::new(code);
+        let mut $machine = Machine {
+            system: &mut system,
+            state: &mut state,
+            bytecode: &bytecode,
+            pc: 0,
+            output: Output::default(),
+        };
+
+        $($body)*
+    };
+    (($rt:ident: $($init:tt)*) ($machine:ident) [ $($inst:tt;)* ] $($body:tt)*) => {
```

Comment:
i made syntax to look like `(rt: <init>) (m) [..] {<body>}` instead of `(rt) { <init> } (m) [..] {<body>}` because i want to be able to access variables in the init section and rust convention of blocks has the user assume that local variables a dropped.